public/documentation/download_mode.md

```diff
@@ -0,0 +1,106 @@
+# Using the Crosswalk Project Runtime in Download Mode
+
+Download Mode provides another way to shrink the application size just like what shared mode does. Your application is still bundled with xwalk_shared_library which contains only the API layer of crosswalk, while the crosswalk core libraries/resources will be downloaded in the background at the first launch of your application from your specified server. Compared to shared mode, the crosswalk runtime is exclusive to your application therefor you are on control of the crosswalk runtime full lifecycle, and the crosswalk runtime download in the background could be done silently without any end user interaction.
+
+# How to enable Download Mode with Embedding API
+
+## Initialize crosswalk with XWalkInitializer/XWalkUpdater
+
```

Comment:
Some explanation of what the code does is usually good to have before including a bunch of code.  Is this correct:
 1) The developer has to create the Crosswalk app as if using the shared library when they ship their application, 
 2) The developer puts a meta tag in the AndroidManifest.xml to say "this is not a shared-mode app, it is a "Download mode" app"
If that is correct, I don't think it is explained.  If that is not correct then I'm confused :)